app/src/components/Footer/index.tsx

```diff
@@ -43,16 +50,20 @@ function FooterBase({ className, style }: FooterProps) {
           target="_blank"
           rel="noreferrer"
         >
-          <IconSpan>
-            <FiberManualRecord className="point" />{' '}
-            {network.name.toLowerCase().indexOf('mainnet') !== 0 && (
-              <b>[{network.name.toUpperCase()}] </b>
-            )}
-            Latest Block: {lastSyncedHeight}
-          </IconSpan>
+          <BlockInfo
+            chainName={Chain.Terra}
+            networkName={network.name}
+            blockNumber={lastSyncedHeight}
+          />
         </a>
 
-        <Link to="/terms">Terms</Link>
+        {isEVM && evmBlockNumber && (
+          <BlockInfo blockNumber={evmBlockNumber} chainName={chain} />
```

Comment:
I think in this case we should create an EvmBlockInfo component which just returns BlockInfo but also encapsulates the useEvmBlockNumber hook.. this way we can stop the EVM block number query from occurring on the terra network as the hook will only run when that EvmBlockInfo component is executed.

Reply:
✅ https://github.com/Anchor-Protocol/anchor-web-app/pull/304/commits/cbe77e40d8974f85295f0ddf5352fbcf09671a69